### DISCUSSION TITLE
Apakah AdBlockID Menangani CAPTCHA?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1748] [Q&A] [answered]
### Sample URL

https://vid.kimcilonlyofc.site/cewek-cantik-ngewe-telen-pejuh-pacar/?player=2

### Ad Blocker

AdGuard

### Keterangan

![IMG_20240415_161841](https://github.com/realodix/AdBlockID/assets/114223791/a0024eb9-62df-47d9-912b-f8c935d6d9ff)

**Comment by realodix** [comment, marked as answer]
Halo, AdBlockID tidak menangani CAPTCHA
